### PR TITLE
Doc aggregation

### PR DESCRIPTION
**Pull request by nilshempelmann**
Update of birdhouse framework
Proposition to aggregate Pavics documentaion in architecture section

### CODE REVIEW

File: docs/source/overview.rst [hunk 1]
```diff
@@ -11,15 +11,15 @@ Overview
...
+.. _documentation_structure
+
+Documentation structure
+-----------------------
 
-The documentation of birdhouse is representing its nature of separate software components and therefor organized in separate documentations. Birdhouse is being used by international working groups pitching up projects with individual focus on the projects objectives. The following graph shows an overview of the documentations organization and how they are aggregated to each other for a better understanding of where to find which information:
+The documentation of birdhouse is representing its nature of separate software components and therefor organized in separate documentations according to the :ref:`birdhouse framework structure <framework_structure>`. Birdhouse is being used by international working groups pitching up projects with individual focus on the projects objectives. The following graph shows an overview of the documentations organization and how they are aggregated to each other for a better understanding of where to find which information:
```
Comment (huard) [inline docs/source/overview.rst]: The documentation of birdhouse aggregates separate documentation from multiple components (:ref:`birdhouse framework structure <framework_structure>`). Individual components are development and maintained by  international working groups motivated to advance specific objectives. To help finding where to look for information, the following graph shows an overview of the documentations organization and how they are connected to each other :

File: docs/source/projects.rst [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
+Architectures examples
+======================
 
 
 .. contents::
     :local:
     :depth: 3
 
-Birdhouse is used in the following projects:
+The birdhouse :ref:`framework is modular organized <framework_structure>` to enable a flexible architecture design depending on the projects needs. Due to the OCG Standard, software components non-birdhouse components can be combined for interoperability. Here are some examples of real projects to show the flexibility and potential of the birdhouse framework.
```
Comment (huard) [inline docs/source/projects.rst]: The birdhouse :ref:`framework is modular <framework_structure>` to enable a flexible architecture design based on individual projects' needs. The reliance on OCG standards allows independent software components to be combined and interoperate. Here are some examples of real projects that show the flexibility and potential of the birdhouse framework.

File: docs/source/overview.rst (hunk 1 above)
Comment (Zeitsperre) [reply docs/source/overview.rst]: The Birdhouse documentation is built from separate documentations stemming from its various components (:ref:birdhouse framework structure <framework_structure>). Birdhouse is an internationally-coordinated effort comprising working groups motivated to advance specific objectives. As such, individual components are developed and maintained by several different working groups. To aid you in knowing where to look for specific information, the following graph portrays an overview of the documentations organization and how the many building blocks of Birdhouse are connected to each other :

**Comment by huard** [comment]
@nilshempelmann Did you include my edits ? I just included them as comments, I don't think they're committed.

**Comment by nilshempelmann** [comment]
Oh, I didn't got that. 
How do I fix that now? I made already a revert branch, is this the right way to go?

**Comment by huard** [comment]
Just push to master...

**Comment by nilshempelmann** [comment]
Done.

**Pull request merged**